kie-spring-boot/kie-spring-boot-samples/kie-server-spring-boot-sample/src/test/java/org/kie/server/springboot/samples/KieServerMigrationTest.java

```diff
@@ -122,8 +126,12 @@ public void setup() {
     
     @After
     public void cleanup() {
-        kieServicesClient.disposeContainer(containerId);
-        kieServicesClient.disposeContainer(containerId2); 
+        if (kieServicesClient != null) {
+            ServiceResponse<Void> response = kieServicesClient.disposeContainer(containerId);
+            logger.info("Container {} disposed with response - {}", containerId, response.getMsg());
+            kieServicesClient.disposeContainer(containerId2);
```

Comment:
Missing response assignation to be traced in the next line (if not, there's the response from the first disposal)
```suggestion
            response = kieServicesClient.disposeContainer(containerId2);
```

Reply:
Thanks! missed that. Done